Content.Shared/Weapons/Ranged/Systems/SharedGunSystem.cs

```diff
@@ -414,7 +414,7 @@ public abstract void Shoot(
         EntityUid? user = null,
         bool throwItems = false);
 
-    public void ShootProjectile(EntityUid uid, Vector2 direction, Vector2 gunVelocity, EntityUid gunUid, EntityUid? user = null, float speed = 20f)
+    public void ShootProjectile(EntityUid uid, Vector2 direction, Vector2 gunVelocity, EntityUid? gunUid = null, EntityUid? user = null, float speed = 20f)
     {
         var physics = EnsureComp<PhysicsComponent>(uid);
```

Comment:
Is null gunuid even used anywhere.

Reply:
Should I remove the default argument and reintroduce the `null` in ProjectileGrenadeSystem?